<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushBtn_left">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>444</y>
     <width>88</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtn_right">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>444</y>
     <width>88</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtn_down">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>488</y>
     <width>88</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtn_up">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>400</y>
     <width>88</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtn_ICON">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>100</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
